person.rb

```diff
@@ -1,9 +1,12 @@
-class Person
-  attr_accessor :name, :age, :parent_permission
+require './nameable'
+
+class Person < Nameable
   attr_reader :id
+  attr_accessor :name, :age
 
   def initialize(age, name = 'Unknown', parent_permission: true)
-    @id = Random.rand(1..100)
+    super()
+    @id = Random.rand(1..1000)
```

Comment:
- [ ] Kindly make sure that there is a method `correct_name` in person.rb . As you don't have this currently the following code to decorate a class cant run it have errors:
```ruby
person = Person.new(22, 'maximilianus')
 person.correct_name
 capitalizedPerson = CapitalizeDecorator.new(person)
 capitalizedPerson.correct_name
 capitalizedTrimmedPerson = TrimmerDecorator.new(capitalizedPerson)
 capitalizedTrimmedPerson.correct_name
```

- [ ] Kindly make sure this code runs correctly before proceeding with another review.